content/applications/finance/accounting/bank/feeds/enablebanking.rst

```diff
@@ -0,0 +1,42 @@
+===============================================
+Enable Banking as bank synchronization provider
+===============================================
+
+**Enable Banking** is a third party provider aggregating banking information from bank accounts all
+in one place.
+
+.. image:: enablebanking/enablebanking.png
+   :align: center
+   :alt:   Enable Banking logo
+
+**Enable Banking** offers non-intrusive connectivity to ASPSPs’ official APIs across Europe without
+storing data. Odoo synchronizes directly with banks to get access to all bank transactions and
+automatically import them into your database.
+
+Configuration
+=============
+
+Link bank accounts with Odoo
+----------------------------
+
+In Odoo, all bank transactions are directly synchronized and imported automatically into your
```

Comment:
I would say: "and automatically imported" to match the order of "are directly synchronized"